README.md

```diff
@@ -2,4 +2,84 @@
 
 [![Build Status](https://travis-ci.org/SUSE/helm-certgen.svg?branch=master)](https://travis-ci.org/SUSE/helm-certgen)
 
-This is a plugin for [Kubernetes Helm Cli](https://github.com/kubernetes/helm) which enables certificate generation for helm charts.
+This repository is for `certgen` plugin for [Kubernetes Helm](https://github.com/kubernetes/helm) CLI.
+This plugin enables TLS certificate generation for helm charts. 
+
+Helm chart that comes with cert.yaml file, which outlines the requirements around certificates,
+this plugin can be used to generate those certificates and place them as a Kubernetes secret in the appropriate namespace.
```

Comment:
Can this be rephrased to make it a little more clear? 
"To use the helm-certgen plugin, the helm charts should have a cert.yaml file which outlines the requirements for certificates (the CSR details, alias names etc.,). The plugin will use these information to generate the certificates and place them as a Kubernetes secret in the appropriate namespace."